build_tools/python/e2e_test_framework/models/torch_models.py

```diff
@@ -12,10 +12,10 @@
 import e2e_test_framework.models.utils as model_utils
 
 # Implementations of the models listed below can be found in
-# https://github.com/iree-org/iree-samples/tree/main/iree-torch/importer.
+# https://github.com/iree-org/iree-experimental/tree/main/iree-torch/library/models.
 # We import the PyTorch models offline and make the .mlir available here for benchmarking.
 # If the mlir artifacts need to be updated, please run
-# https://github.com/iree-org/iree-samples/blob/main/iree-torch/importer/update_torch_models.sh
+# https://github.com/iree-org/iree-experimental/blob/main/iree-torch/library/import_torch_models.sh
 # Then update the `source_url` below with the new paths.
```

Comment:
Note that we still have these tests disabled: https://github.com/openxla/iree/issues/14993

https://github.com/openxla/iree/blob/8711c812455955f9dc252dba0d7943a7dba15cbe/build_tools/python/e2e_test_framework/models/model_groups.py#L147-L161